presto-main/src/main/java/com/facebook/presto/sql/planner/PlanPrinter.java

```diff
@@ -211,11 +220,18 @@ public static String textDistributedPlan(List<StageInfo> stages, Metadata metada
         // it's possible that some or all of them are missing or out of date.
         // For example, a LIMIT clause can cause a query to finish before stats
         // are collected from the leaf stages.
-        Map<PlanNodeId, Long> inputPositions = new HashMap<>();
-        Map<PlanNodeId, Long> inputBytes = new HashMap<>();
-        Map<PlanNodeId, Long> outputPositions = new HashMap<>();
-        Map<PlanNodeId, Long> outputBytes = new HashMap<>();
-        Map<PlanNodeId, Long> wallMillis = new HashMap<>();
+        Map<PlanNodeId, Long> planNodeInputPositions = new HashMap<>();
+        Map<PlanNodeId, Long> planNodeInputBytes = new HashMap<>();
+        Map<PlanNodeId, Long> planNodeOutputPositions = new HashMap<>();
+        Map<PlanNodeId, Long> planNodeOutputBytes = new HashMap<>();
+        Map<PlanNodeId, Long> planNodeWallMillis = new HashMap<>();
+
+        Map<PlanNodeId, Map<String, Long>> operatorTotalDrivers = new HashMap<>();
```

Comment:
When is it helpful to know the total number of drivers? I think it would be simpler to print this more like:
```
- InnerJoin[("orderkey" = "orderkey_0")][$hashvalue, $hashvalue_7] => [orderkey:bigint, $hashvalue:bigint, orderkey_0:bigint, $hashvalue_7:bigint] 
        Cost: 32.52%, Output: 6001215 rows (206.04MB)                                                                                              
        Left (probe) input avg.: 1500303.75 lines, Input std.dev.: 0.03%                                                      
        Right (build) input avg.: 93750.00 lines, Input std.dev.: 0.23%                                                      
                Collisions avg.: 117186.29 (99.60% est.), Collisions std.dev.: 1.56%
```